lib/trino-arrow/pom.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If the arrow classes are only meant to be used in the bigquery connector, they should be added there instead of creating a new module. If later we need to reuse them, we can pull them out into a separate module. Let's not over-engineer prematurely.